editors/vscode/package.json

```diff
@@ -110,8 +110,8 @@
         "bin/slint-lsp-*"
     ],
     "scripts": {
-        "vscode:prepublish": "npm run compile",
-        "compile": "wasm-pack build --target web --out-name index ../../tools/lsp --no-default-features --features preview-lense && node ./esbuild.js && wasm-pack build --release --target web --out-dir $PWD/out ../../api/wasm-interpreter",
+        "vscode:prepublish": "wasm-pack build --target web --out-name index ../../tools/lsp --no-default-features --features preview-lense && wasm-pack build --release --target web --out-dir $PWD/out ../../api/wasm-interpreter",
+        "compile": "node ./esbuild.js",
```

Comment:
:+1:   because this will indeed speedup a lot compilation when testing locally.  (When i do debugging myself, i disable wasm-opt, so it stays fast for incremental update)

But you need to update the README because compiling locally will need another command.

(Ideally it'd be nice if there was two command: one to build a local-only extension, and one to build the local+web)

Reply:
ah yes, the instruction don't mention npm compile. It seems it's just what i'm using.